### PR TITLE
[PHP] Add Webman

### PR DESCRIPTION
**Pull request by walkor**
Add [Webman](https://github.com/walkor/webman)

Webman is a high performance HTTP Service Framework for PHP based on Workerman.

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
Hi @walkor! Could you remove all the files that aren't necessary for this benchmark to run like example `.env`s and other configuration that isn't used

**Comment by walkor** [comment]
OK, I'm trying to get rid of the useless files.

**Comment by walkor** [comment]
done.

File: frameworks/PHP/webman/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "language": "PHP",
+      "flavor": "PHP7",
+      "orm": "Raw",
+      "platform": "workerman",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "webman",
+      "notes": "",
+      "versus": "php"
```
Comment (joanhey) [inline frameworks/PHP/webman/benchmark_config.json:25]: It's better compare with workerman.
So you will know the framework overhead.

Comment (walkor) [reply frameworks/PHP/webman/benchmark_config.json:25]: Do you mean to change `"display_name": "webman"` to `"display_name": "webman-workerman"` ?

Comment (walkor) [reply frameworks/PHP/webman/benchmark_config.json:25]: Do you mean to change `"display_name": "webman"` to `"display_name": "webman-workerman"` ?

Comment (joanhey) [reply frameworks/PHP/webman/benchmark_config.json:25]: "versus": "workerman"

Comment (walkor) [reply frameworks/PHP/webman/benchmark_config.json]: OK, thank you for your advice.

**Pull request merged**